[node] Can't augement static side of Module since 13.1.2

Since https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41225 I'm no longer able to augment the static side of `class Module`.  Before it was possible as indicated there: #40927 (comment)

I get `Duplicate identifier 'Module' as error. Any hints how to tell typescript to merge the declarations?

- [x] I tried using the `@types/node` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @SimonSchick @sandersn @andrewbranch

I tried to add `namespace Module {}` in module.d.t and the following in my code (in a dedicated .d.ts file):
```ts
declare module "module" {
  namespace Module {
    function _load(request: string, parent: NodeModule | undefined, isMain?: boolean): any;
    function _resolveFilename(request: string, parent: NodeModule | undefined): string;
  }
}
```

I get following errors:
```
node_modules/@types/node/module.d.ts(15,16): error TS2300: Duplicate identifier 'Module'.
node_modules/@types/node/module.d.ts(20,5): error TS2309: An export assignment cannot be used in a module with other exported elements.
typings/extensions/node-ext.d.ts(85,12): error TS2300: Duplicate identifier 'Module'.
```

In @types/node v12 following works:
```ts
declare namespace NodeJS {
  namespace Module {
    function _load(request: string, parent: NodeModule | undefined, isMain?: boolean): any;
    function _resolveFilename(request: string, parent: NodeModule | undefined): string;
  }
}
```

Still get an error:
```node_modules/@types/node/module.d.ts(20,5): error TS2309: An export assignment cannot be used in a module with other exported elements.
```

--- a/types/node/module.d.ts
+++ b/types/node/module.d.ts
@@ -1,5 +1,6 @@
 declare module "module" {
     import { URL } from "url";
+    namespace Module {}
     interface Module extends NodeModule {}
     class Module {
         static runMain(): void;

